commons/data_access_layer/cosmos_db.py

```diff
@@ -133,6 +133,13 @@ def create_sql_condition_for_visibility(
             return 'AND NOT IS_DEFINED(%s.deleted)' % container_name
         return ''
 
+    @staticmethod
+    def create_sql_condition_for_active( only_active: bool,
```

Comment:
rename  **create_sql_condition_for_active** to **active_condition**

Reply:
I think `create_sql_is_active_condition` would be ok to keep the standard naming present in the code.

Reply:
Ok, it has been adjusted, with a new approach to avoid the only_active argument.